.husky/commit-msg

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
I see that the basics of husky are set up, but do not see any actions performed.